tests/test/test_volunteer_inventory.py

```
@@ -0,0 +1,66 @@
+        checkout_page = CheckOutPage(driver)
+        checkout_page.click_checkout()
+        checkout_page.click_building_code()
+        checkout_page.select_first_building_option()
+        checkout_page.click_unit_number()
+        checkout_page.select_first_unit_number()
+        checkout_page.click_name_input()
+        checkout_page.select_first_unit_number()  # Possibly redundant?
+        checkout_page.click_continue_button()
+        checkout_page.search_item(item)
+        checkout_page.add_item(item)
+        3#checkout_page.items_added(item)
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)